### DISCUSSION TITLE
Add ability to subscribe to MDN Plus plans using USD card from an unsupported country

### DISCUSSION THREAD

**Discussion by bershanskiy** [mdn/mdn-community#122] [MDN Plus] [answered]
I'm from a country not listed among supported countries](https://hacks.mozilla.org/2022/04/mdn-plus-now-available-in-more-markets/). I have a card in USD from a bank in my country and would like to use it to pay for MDN Plus as if I was in US. I tried creating a new Firefox account, but I can not manually choose a region.

Repro steps:
1. Be in a country not on [this list](https://hacks.mozilla.org/2022/04/mdn-plus-now-available-in-more-markets/)
2. Go to https://developer.mozilla.org/en-US/plus#subscribe and see only Core/Free plan
3. Go to Firefox account settings page and observe no way to change region

**Comment by pransh15** [comment, marked as answer]
I'm sorry you can't access the MDN Plus service. I hope you can use MDN for free until we can reach you with MDN Plus.

Closing this for now, feel free to start a new discussion if there's an issue. :)
